### ISSUE TITLE
Missing Entities

### ISSUE THREAD

**Issue by claudermilk** [simbaja/ha_gehome#179]
I've been having trouble with the integration lately. The latest is some entities have vanished from the devices. 

I have two registered right now: GE Cafe Oven and GE Hood. The hood dropped the fan and light entities which were the obvious ones. Looking more, I noticed the oven dropped the new warming drawer entity, and the oven light. 

I've removed and reinstalled the integration a few times over the last couple of days (kept losing connection leaving all states as unknown). I've also redownloaded the add-on. I don't see any messages in the log.

**Comment by claudermilk** [comment]
Quick follow-up. I forgot to mention I'm updated to the latest version of HA: Supervisor 2023.07.1, Core 2023.7.1, OS 10.3.
I also thought getting output from gehome-appliance-data might help, but that's now throwing a command not found. I tried to install again, but got this mess of errors.
Building wheels for collected packages: slixmpp, cffi
  Building wheel for slixmpp (pyproject.toml) ... done
  Created wheel for slixmpp: filename=slixmpp-1.8.4-py3-none-any.whl size=508907 sha256=9d1373af268b25f2c7f429eedbbcded6ac8bc4758a8d3a1fe2821421bceceadd
  Stored in directory: /root/.cache/pip/wheels/5c/57/64/dd9fabb3db37e00482ec34194434c8b916728728dd51a7ca6a
  Building wheel for cffi (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for cffi (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [79 lines of output]
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'libffi', required by 'virtual:world', not found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'libffi', required by 'virtual:world', not found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'libffi', required by 'virtual:world', not found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'libffi', required by 'virtual:world', not found
      Package libffi was not found in the pkg-config search path.
      Perhaps you should add the directory containing `libffi.pc'
      to the PKG_CONFIG_PATH environment variable
      Package 'libffi', required by 'virtual:world', not found

          No working compiler found, or bogus compiler options passed to
          the compiler from Python's standard "distutils" module.  See
          the error messages above.  Likely, the problem is not related
          to CFFI but generic to the setup.py of any Python package that
          tries to compile C code.  (Hints: on OS/X 10.8, for errors about
          -mno-fused-madd see http://stackoverflow.com/questions/22313407/
          Otherwise, see https://wiki.python.org/moin/CompLangPython or
          the IRC channel #python on irc.libera.chat.)

          Trying to continue anyway.  If you are trying to install CFFI from
          a build done in a different context, you can ignore this warning.

      /tmp/pip-build-env-3ukwkkcg/overlay/lib/python3.11/site-packages/setuptools/config/setupcfg.py:293: _DeprecatedConfig: Deprecated config in `setup.cfg`
      !!

              ********************************************************************************
              The license_file parameter is deprecated, use license_files instead.

              By 2023-Oct-30, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.

              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************

      !!
        parsed = self.parsers.get(option_name, lambda x: x)(value)
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-aarch64-cpython-311
      creating build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/verifier.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/backend_ctypes.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/pkgconfig.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/lock.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/cparser.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/model.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/__init__.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/recompiler.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/api.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/commontypes.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/cffi_opcode.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/vengine_cpy.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/vengine_gen.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/setuptools_ext.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/ffiplatform.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/error.py -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/_cffi_include.h -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/parse_c_type.h -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/_embedding.h -> build/lib.linux-aarch64-cpython-311/cffi
      copying cffi/_cffi_errors.h -> build/lib.linux-aarch64-cpython-311/cffi
      running build_ext
      building '_cffi_backend' extension
      creating build/temp.linux-aarch64-cpython-311
      creating build/temp.linux-aarch64-cpython-311/c
      gcc -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DFFI_BUILDING=1 -I/usr/include/ffi -I/usr/include/libffi -I/usr/include/python3.11 -c c/_cffi_backend.c -o build/temp.linux-aarch64-cpython-311/c/_cffi_backend.o
      error: command 'gcc' failed: No such file or directory
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for cffi
Successfully built slixmpp
Failed to build cffi
ERROR: Could not build wheels for cffi, which is required to install pyproject.toml-based projects

**Comment by claudermilk** [comment]
I poked at it again. The hood light and fan entities and oven light entity are back. I'm not sure why they disappeared at all--I wonder if something on a server hiccuped. The warming drawer entity just added to the integration is still missing. I also tried reloading the gehome-appliance-data and this time it worked. I have output from the devices again, and I see the warming drawer showing there.
